README.rst

```diff
@@ -3,16 +3,16 @@ pypuppetdb
 ##########
 
 .. image:: https://api.travis-ci.org/puppet-community/pypuppetdb.png
-   :target: https://travis-ci.org/puppet-communit/pypuppetdb
+   :target: https://travis-ci.org/puppet-community/pypuppetdb
 
 .. image:: https://coveralls.io/repos/puppet-community/pypuppetdb/badge.png
-   :target: https://coeralls.io/r/puppet-community/pypuppetdb
+   :target: https://coveralls.io/repos/puppet-community/pypuppetdb
+
 
 pypuppetdtb is a library to work with PuppetDB's REST API. It is implemented
 using the `requests`_ library.
 
-**pypuppetdb DOES NOT work with PuppetDB 3**
-
+**pypuppetdb >= 0.2.0 DOES work with PuppetDB 3**
```

Comment:
That expressions means the versions 0.2.0 and LESS work with PuppetDB 3.

This means that versions 0.2.0 and GREATER work with PuppetDB 3